FTB/Signatures/CrashInfo.py

```diff
@@ -1148,75 +1149,88 @@ def __init__(self, stdout, stderr, configuration, crashData=None):
+                # First extract the line
+                # 32-bit example:
+                #     eax=02efff01 ebx=016fddb8 ecx=2b2b2b2b edx=016fe490 esi=02e00310 edi=02e00310
+                # 64-bit example:
+                #     rax=00007ff74d8fee30 rbx=00000285ef400420 rcx=2b2b2b2b2b2b2b2b
                 matchLine = re.search(RegisterHelper.getRegisterPattern(), line)
-                if matchLine != None:
+                if matchLine is not None:
                     ecxrData.extend(line.split())
 
-                # Next, put the rax, rbx, rcx, etc. entries into a list of their own, then iterate
+                # Next, put the eax/rax, ebx/rbx, etc. entries into a list of their own, then iterate
                 match = re.search(cdbRegisterPattern, line)
                 for instr in ecxrData:
                     match = re.search(cdbRegisterPattern, instr)
-                    if match != None:
+                    if match is not None:
                         register = match.group(1)
                         value = long(match.group(2), 16)
                         self.registers[register] = value
 
             # Crash address
-            if line.startswith("Exception Faulting Address:"):
-                # Example:
-                #     Exception Faulting Address: 0x7fef86c13e4
-                address = line.split(": ")[1]
+            if line.startswith("ExceptionAddress:"):
+                # 32-bit example:
+                #     ExceptionAddress: 00404c59 (js_32_dm_windows_62f79d676e0e!JSObject::is+0x00000002)
+                # 64-bit example:
+                #     ExceptionAddress: 00007ff74d469ff3 (js_64_dm_windows_62f79d676e0e!JSObject::is+0x000000000000000a)
+                address = line.split()[1]
                 self.crashAddress = long(address, 16)
 
             # Crash instruction
-            if line.startswith("Faulting Instruction:"):
-                # Example:
-                #     Faulting Instruction:01206fbd int 3
-                cInstruction = line.split(":")[1].split(None, 1)[1]
+            # 64-bit binaries have a backtick in their addresses, e.g. 00007ff7`1e424e62
+            lineWithoutBacktick = line.replace("`", "")
+            if address and lineWithoutBacktick.startswith(address):
+                # 32-bit example:
+                #     00404c59 8b39            mov     edi,dword ptr [ecx]
+                # 64-bit example:
+                #     00007ff7`4d469ff3 4c8b01          mov     r8,qword ptr [rcx]
+                cInstruction = line.split(None, 2)[-1]
+                # There may be multiple spaces inside the faulting instruction
+                cInstruction = " ".join(cInstruction.split())
                 self.crashInstruction = cInstruction
 
             # Start of stack for crashing thread
-            if re.match(r'^\s*Hash Usage : Stack Trace:', line):
+            if line.startswith("STACK_TEXT:"):
```

Comment:
Is this a Win7->Win10 change or is that just a generic improvement? Does it still work with the older traces?